lib/cdo/honeybadger.rb

```diff
@@ -84,4 +84,30 @@ def self.parse_exception_dump(error)
 
     [error_message, error_lines]
   end
+
+  # Returns all issues (across cronjobs, dashboard, and pegasus) that have occured since midnight.
+  # @return [Array[Hash]] An array of hashes summarizing the recent issues.
+  def self.get_recent_issues
+    issues = []
+
+    midnight_epoch = Time.now.to_i / 86400 * 86400
```

Comment:
This was a "random" choice. To what extent do we expect the previous DOTD to deal with stuff happening overnight or before the handoff? Feel welcome to recommend a different cutoff.